x/application/types/message_unstake_application.go

```diff
@@ -1,8 +1,8 @@
 package types
 
 import (
+	sdkerrors "cosmossdk.io/errors"
```

Comment:
NIT: Not major but in places we import this as `errorsmod` and others `sdkerrors` I am not aliasing it and just calling it `errors` (we dont use native `errors` package as this does wrapping for us so basically multierr too).

Just think consistency here would be better but not a huge deal